src/material/datepicker/date-range-input.ts

```diff
@@ -154,7 +155,7 @@ export class MatDateRangeInput<D> implements MatFormFieldControl<DateRange<D>>,
 
     if (newValue !== this._groupDisabled) {
       this._groupDisabled = newValue;
-      this._disabledChange.next(this.disabled);
+      this._stateChanges.next();
```

Comment:
Because of upcoming rxjs changes we always want to be passing something to `next` now, e.g. `next(null)`

Reply:
I changed it to `next(undefined)` since some of these are implemented due to `MatFormFieldControl` which requires a `stateChanges: Observable<void>`.